docs/getting-started/smr-disk.md

```diff
@@ -115,24 +115,25 @@ sd 5:0:0:0: [sdb] Attached SCSI disk
 ...
 ```
 
-Among other information, the zone model of the disk is confirmed as
-host managed. The total number of zones of the disk is also displayed. In this
-example, the disk capacity is 15 TB and has 55880 zones.
+The kernel ring buffer message " sd 5:0:0:0: [sdb] Host-managed zoned block device" confirms that the zone model of the disk is "host managed". 
```

Comment:
Same here. We do not need "ring buffer".

Reply:
I've removed it.